src/algorithms/calorimetry/ImagingTopoCluster.h

```diff
@@ -65,7 +66,7 @@ namespace eicrecon {
 
     // unitless counterparts of the input parameters
     double localDistXY[2]{0, 0}, layerDistEtaPhi[2]{0, 0}, layerDistXY[2]{0, 0}, sectorDist{0};
```

Comment:
Two comments (not pertaining to this PR): these should all be defined on their own line. And if we really want to use arrays here we should at least use `std::array<double, 2>` instead of raw C-arrays.

Reply:
714aec408